podcast.rss

```diff
@@ -7341,5 +7341,349 @@
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/5-vqd-podcast-unit78.mp3" type="audio/mp3" length="2895281"/>
+      <itunes:duration>12:42</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>5-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>631.സൂറഃ നബ ആയത്ത് 6_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #631 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; 6* أَلَمْ نَجْعَلِ ٱلْأَرْضَ مِهَٰدًۭا ഭൂമിയെ നാം ഒരു വിരിപ്പ്&zwnj; (മെത്ത) ആക്കിയില്ലേ? 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/6-vqd-podcast-unit78.mp3" type="audio/mp3" length="2758339"/>
+      <itunes:duration>13:14</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>6-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>632.സൂറഃ നബ ആയത്ത് 7_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #632 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; 7* وَٱلْجِبَالَ أَوْتَادًۭا മലകളെ ആണികളും? 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/7-vqd-podcast-unit78.mp3" type="audio/mp3" length="1709514"/>
+      <itunes:duration>07:56</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>7-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>633.സൂറഃ നബ ആയത്ത്  8_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #633 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; 8* وَخَلَقْنَٰكُمْ أَزْوَٰجًۭا നാം നിങ്ങളെ ഇണകളായി സൃഷ്ടിച്ചു. 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/8-vqd-podcast-unit78.mp3" type="audio/mp3" length="2798513"/>
+      <itunes:duration>12:23</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>8-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>634.സൂറഃ നബ ആയത്ത് 9 Part 1_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #634 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; -9* വിവരണം - 1️⃣ وَجَعَلْنَا نَوْمَكُمْ سُبَاتًۭا നിങ്ങളുടെ ഉറക്കത്തെ നാം വിശ്രമമാക്കുകയും ചെയ്തു. 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/9-vqd-podcast-unit78.mp3" type="audio/mp3" length="2924441"/>
+      <itunes:duration>13:00</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>9-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>635.സൂറഃ നബ ആയത്ത് 9 Part 2_a</title>
```

Comment:
@febincp6 title issue here again. Can we get rid of all **_a** from the title? Pls consider all titles too.

Reply:
rs